esp/src/src-react/components/forms/Fields.tsx

```diff
@@ -902,6 +902,7 @@ export function createInputs(fields: Fields, onChange?: (id: string, newValue: a
                     label: field.label,
                     field: <input
                         key={fieldID}
+                        id={fieldID}
                         type="datetime-local"
                         name={fieldID}
                         defaultValue={field.value}
```

Comment:
Would changing "defaultValue" here to "value" have fixed the issue also?

Reply:
It did not. Using "value" instead, the filter would have updated with the selected datetime on submit, but the input wouldn't display it.